test/Microsoft.AspNet.Mvc.FunctionalTests/Compiler/Resources/MvcTagHelpersWebSite.MvcTagHelperTest.Order.html

```diff
@@ -0,0 +1,79 @@
+﻿
+
+
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title></title>
+</head>
+<body>
+    <form action="/Order/Submit" method="post">
+        <div>
+            <label class="order" for="Shipping">Shipping</label>
+            <input size="50" type="text" id="Shipping" name="Shipping" value="Your shipping method is UPSP" />
+        </div>
+        <div>
+            <label class="order" for="ShippingDateTime">ShippingDateTime</label>
+            <input type="datetime-local" id="ShippingDateTime" name="ShippingDateTime" value="1/1/0001 12:00:00 AM" />
+        </div>
+        <div>
+            <label class="order" for="Products">Products</label>
+            <select Multiple="multiple" id="Products" name="Products"><option value="7">Product_7</option>
+<option value="8">Product_8</option>
+<option selected="selected" value="9">Product_9</option>
+<option value="10">Product_10</option>
+<option value="11">Product_11</option>
+<option value="12">Product_12</option>
+</select>
+        </div>
+        <div>
+            <label class="order" for="OrderDate">OrderDate</label>
+            <input type="datetime" id="OrderDate" name="OrderDate" value="0001/01/01/ A.D." />
+        </div>
+        <div>
+            <label class="order" for="NeedSpecialHandle">NeedSpecialHandle</label>
+            <input checked="checked" id="NeedSpecialHandle" name="NeedSpecialHandle" type="checkbox" value="true" /><input name="NeedSpecialHandle" type="hidden" value="false" />
+        </div>
+        <div>
+            <label class="order" for="PaymentMethod">PaymentMethod</label>
+            <select id="PaymentMethod" multiple="multiple" name="PaymentMethod">
+                <option Value="Credit">Credit</option>
+                <option Value="Check" selected="selected">Check</option>
+            </select>
+        </div>
+        <div>
+            <label class="order" for="Customer_Number">Number</label>
+            <input class="form-control" type="number" data-val="true" data-val-range="The field Number must be between 1 and 100." data-val-range-max="100" data-val-range-min="1" id="Customer_Number" name="Customer.Number" value="1" />
+            <span class="field-validation-valid" data-valmsg-for="Customer.Number" data-valmsg-replace="true"></span>
```

Comment:
one more teest: need to submit invalid data and confirm the `class` attributes of the affected `<span/>`s are correct